stable/aws-for-fluent-bit/templates/psp.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.rbac.pspEnabled }}
```

Comment:
@razorsk8jz are you sure that you want to wrap this entire file with this option? I think that means this existing file will not get generated unless you have this new default false field enabled. That seems non-desirable. 

Can you please explain the intention behind this? Should the option be default true?

I also see here that pod security policy is removed in newer k8s version: https://kubernetes.io/docs/concepts/security/pod-security-policy/

So I'm not sure it makes sense to move forward with this.

Reply:
I see so the intention is to make it default false because k8s has removed pod security policy: https://github.com/aws/eks-charts/issues/810

Reply:
The chart seems to currently have this which defaults false, shouldn't this be used to gate creating this file or not?? https://github.com/fluent/helm-charts/blob/main/charts/fluent-bit/values.yaml#L35

Reply:
Sorry I'm really a novice at some of the things, I need you to explain it to me.

Reply:
I understand now, this makes it disabled by default, which we want since its deprecated.